pandas/core/series.py

```diff
@@ -3441,8 +3441,10 @@ def sort_values(
                 )
             ascending = ascending[0]
 
-        if not is_bool(ascending):
-            raise ValueError("ascending must be boolean")
+        #if not is_bool(ascending):
+        #    raise ValueError("ascending must be boolean")
+        if type(ascending)!=bool:
+            raise ValueError("ascending should of type bool")
```

Comment:
are there some words left out of this message?  what's wrong with the existing one?